packages/generator-particle/src/generators/storybook/templates/main.ts

```diff
@@ -0,0 +1,51 @@
+export interface MainConfig {
+  componentLibraryPath: string // this will have to be resolved based off storybook location in apps/storybook and the path of the component_library
+  addons: string[]
+  storiesRoot: string[]
+}
+
+const stringifyAndSingleQuote = (val: string[]) =>
+  JSON.stringify(val).replace('"', "'")
+
+/**
+ * app/storybook/main.js
+ */
+export const main = (
+  config: MainConfig
+) => `const path = require('path')frontendFramework
+
+const APP_COMPONENT_LIBRARY = path.resolve(__dirname, '../../src/default')
```

Comment:
This is not a given. CLI should be collecting these paths on generation as a project may opt to name their codebase something other than `default`, and there may be multiple front-end codebases in `src` to handle "multi-design-system" use.

Reply:
Agreed. We need a TODO for this. Forgot to add it here. For now I can make a default value of 'default'